input/pagecontent/authentication.md

```diff
@@ -0,0 +1,279 @@
+### Introduction
+
+Authentication is the process of verifying the identity of an entity. In the context of healthcare, authentication is used to verify the identity of healthcare professionals and care organizations. This is important to ensure that only authorized entities can access sensitive information and resources.
+
+The verified identity is often used downstream for authorization and accounting. Authorization is the process of granting access to resources based on the identity of an entity. Accounting is the process of tracking the actions of an entity.
+
+#### Requirements
+
+There exists a multitude of authentication solutions, therefore it is important to define the requirements for authentication in the context of healthcare. The requirements are:
+
+- Work with identity claims from the authoritative sources
+- Support combinations of identity claims from different trusted issuers
+- Support for use-cases with and without an end-user (healthcare professional)
+- Support for authorizing vendors to act on behalf of a (care) organization
+- Support for limiting the scope of a vendor's authorization
+- Support for authorizing (care) organizations to act on behalf of a healthcare professional
+- User-friendly solution for healthcare professionals
+- Cost-efficient solution for care organizations
+- Flexible solution that can be adapted to different use-cases
+- Peer-2-peer trust which allows direct interactions between the involved parties without involvement of a third party
+- No single point of failure
+- Privacy by design
+- Security by design
+- Support for leveraging existing identity solutions
+
+#### Terminology
+
+Throughout this document the following terminology is used:
+
+- Entity: An actor in the system (person or organization)
+- Agent: A digital representation of an entity that acts on its behalf, also often called an identity wallet
+- Vendor: A party that offers Agents to its customers (care organizations and healthcare professionals)
+- Identity: A set of claims about an entity (person or organization) which is relevant in a specific context
+- Verifiable Identifier (VID): An identifier that can be (cryptographically) verified
+- Claim: A statement about an entity (e.g. name, role, affiliation)
+- Verifiable Claim: A claim that can be cryptographically verified
+- Issuer: An entity that issues claims about another entity
+- Authoritative source: An issuer that is trusted to issue claims about a specific entity
+- Verifier: An entity that verifies the identity of another entity based on its claims
+- Holder: An entity that holds claims about itself or is authorized to present claims about another entity
+- Authentication: The process of verifying the identity of an entity
+- Authorization: The process of granting access to resources based on the identity of an entity
+- Use-case: A specific scenario in which authentication and authorization are required
```

Comment:
If we use common terminology defined elsewhere, let's refer to that external source